### PR TITLE
Added the flow on how docs get published and fixed npm inistall docusaurus in README

### LINKED ISSUE #198: Update README.md file
## BUG REPORT

**What happened**:
After following the README to setup openebs-docs on local system, it throws an error
`Error: No sidebar file available to use for version latest. Verify that 'version-latest-sidebars.json' exists`

**What you expected to happen**:
It should start the server.

**How to reproduce it (as minimally and precisely as possible)**:
Clone the repository.
Change directory to openebs-docs/website
Run `npm install docusaurus`
Run `npm start`

Remove `npm install docusaurus` from the README.md file.

### PR DESCRIPTION
**Pull request by sayak119**
Signed-off-by: sayak119 <sk.sayakkundu1997@gmail.com>

fixes #198 and #108 

Please review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,11 +38,24 @@ cd openebs-docs
...
 
-openebs-docs/website$ npm install docusaurus
-
 openebs-docs/website$ npm start
 ```
 Above step will start server on the localhost:3000
+
+### How OpenEBS-docs get published?
+
+These are the steps that happen (from selecting an issue to publishing).
```
Comment (mahebbar) [inline README.md]: The following procedure lists the tasks from the time you select an issue to publishing the document.

File: README.md [hunk 2]
```diff
@@ -38,11 +38,24 @@ cd openebs-docs
...
+
+1. Go through the [issues](https://github.com/openebs/openebs-docs/issues/) and select an issue you want to work on.
+
+2. Go to `openebs-docs/website` and execute `npm start`. You can then preview the doc at `http://localhost:3000/docs/next/quickstartguide.html`.
```
Comment (mahebbar) [inline README.md]: doc = document

File: README.md [hunk 3]
```diff
@@ -38,11 +38,24 @@ cd openebs-docs
...
+
+3. Now work on your issue and submit your pull request(PR) for the members to review. Do perform the DCO signoff. DCO stands for Developer Certificate of Origin. It requires the commit message to have a `Signed-off-by:` statement along with the email address of the author of that commit. You can do this using the following command `git commit -s -m 'Commit message related to the issue'`. You can read more about it [here](https://github.com/probot/dco#how-it-works).
```
Comment (mahebbar) [inline README.md]: Remove "Now"
submit your pull request = create and submit your pull request

File: README.md [hunk 4]
```diff
@@ -38,11 +38,24 @@ cd openebs-docs
...
+
+4. Make any changes to your pull request as suggested by the members. In order to keep the pull request clean you can use `git commit --amend -s -m 'Commit message related to the issue'` along with `git push -f`. This will prevent multiple commits.
```
Comment (mahebbar) [inline README.md]: Make changes

File: README.md [hunk 5]
```diff
@@ -38,11 +38,24 @@ cd openebs-docs
...
+
+5. After you submit your pull request and is approved by atleast one member, it goes for `Travis CI` integration. Your pull request is checked and if it exits with code 0 in all the cases, then its considered passed and good for merging. If it fails, then identify the errors and work on it and resubmit the pull request. You can use the commands mentioned in point 4.
```
Comment (mahebbar) [inline README.md]: considered as "passed"

**Comment by NisantaSahoo** [comment]
Congratulations for your PR getting merged! Thanks for taking part in OpenEBS Hackfest.
Fill up the doc form to get the Hackfest T-Shirt and Swags
https://docs.google.com/spreadsheets/d/1TZYIdERnD72YPeUsdS5hDfI6xgXXCax4-DHpXh-o6r0/edit?usp=sharing
Looking for some more awesome contributions from you and win exciting prizes

**Pull request merged**